.gitignore

```diff
@@ -1,3 +1,3 @@
 .idea/
 target
-
+Cargo.lock
```

Comment:
Why did you add `Cargo.lock` to `.gitignore`?

Reply:
it's useless，I will remove it